tests/run_tests.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Please remove bash. We don't want to maintain bash scripts anymore.

If you feel lazy, convert it to plumbum; otherwise to regular python. And run it inside pytest.

Reply:
Yes. these were the old tests from https://github.com/Tecnativa/docker-socket-proxy/pull/14, I just left them for reference and forgot to remove them. Already done.